environment.py

```diff
@@ -545,4 +545,16 @@ def env() -> Mapping[str, Optional[str]]:
         # https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/account_alternate_contact
         #
         'azul_security_contact': None,
+
+        # Slack integration with AWS Chatbot app is disabled by default, enable
+        # it by replacing the None values of ``azul_slack_integration``, that is
+        # workspace_id and channel_id. The former, is the Slack workspace ID
+        # integrating with the AWS Chatbot app. The latter, is the Slack channel
+        # ID receiving the AWS Chatbot app notifications. See section 3.1.3 of
+        # the README for details.
+        #
+        'azul_slack_integration': json.dumps({
+            'workspace_id': None,
+            'channel_id': None
+        })
```

Comment:
https://github.com/DataBiosphere/azul/pull/4768#discussion_r1049996302

This suggestion from Hannes seems to have been overlooked or rolled back. I would rewrite this entire section as follows:

```suggestion

        # To enable Slack integration with AWS Chatbot, populate this variable with
        # JSON containing the IDs of the Slack workspace and Slack channel for
        # the Chatbot to post notifications to, e.g.:
        # 'azul_slack_integration': json.dumps({
        #     'workspace_id': 'your-workspace-id',
        #     'channel_id': 'your-channel-id' 
        # }) 
        'azul_slack_integration': None
```

I think the variable names are sufficiently self-explanatory that we don't need any more text than that,